sui/wormhole/Move.toml

```diff
@@ -1,14 +1,12 @@
 [package]
 name = "Wormhole"
 version = "0.2.0"
+published-at = "0xcc029e2810f17f9f43f52262f40026a71fbdca40ed3803ad2884994361910b7e"
```

Comment:
The address I see in the testnet branch is `0xf47329f4344f3bf0f8e436e2f7b485466cff300f12a166563995d3888c296a94`

Reply:
That's interesting. Indeed, I changed `published-at` and `wormhole` to
`0xf47329f4344f3bf0f8e436e2f7b485466cff300f12a166563995d3888c296a94`
in `Move.toml` and source verification passed too. Begs the question of
why there are two different deployments with the exact same source code.

In fact, at Aftermath we've been using [`0xcc029e2810f17f9f43f52262f40026a71fbdca40ed3803ad2884994361910b7e`](https://suiexplorer.com/object/0xcc029e2810f17f9f43f52262f40026a71fbdca40ed3803ad2884994361910b7e?network=testnet)
to verify Wormhole messages (and there's more activity in the contract
than just ours looking at the txs.)

Would be good to clarify what's the official version.